SPECS/containerd2/containerd2.spec

```diff
@@ -0,0 +1,86 @@
+%global debug_package %{nil}
+%define upstream_name containerd
+%define commit_hash 207ad711eabd375a01713109a8a197d197ff6542
+
+Summary: Industry-standard container runtime
+Name: %{upstream_name}2
+Version: 2.0.0
+Release: 1%{?dist}
+License: ASL 2.0
+Group: Tools/Container
+URL: https://www.containerd.io
+Vendor: Microsoft Corporation
+Distribution: Azure Linux
+
+Source0: https://github.com/containerd/containerd/archive/v%{version}.tar.gz#/%{upstream_name}-%{version}.tar.gz
+Source1: containerd.service
+Source2: containerd.toml
+
+%{?systemd_requires}
+
+BuildRequires: git
```

Comment:
are we sure? this suggests to me that there may be an online dependency but we build entirely offline. For what step is git required?

Reply:
Good catch, I removed git which is not needed, validated by buddybuild [693469](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=693469&view=results).